### DISCUSSION TITLE
What are .quarto folders, and how can I ignore them or prevent them from being created?

### DISCUSSION THREAD

**Discussion by luifrancgom** [quarto-dev/quarto-cli#9192] [Q&A] [answered]
### Description

I'm curious about `.quarto` folders:

- What's their purpose?
- Why does my project generate them?
- Can I safely ignore them, or is there a way to stop their creation?"

When I render an HTML document, Quarto creates a `.quarto` folder with these subfolders:

-  `_freeze`
-  `xref`

![Screenshot 2024-03-25 183954](https://github.com/quarto-dev/quarto-cli/assets/50002526/3b78cb70-7b0f-4151-b41a-ed38ad648d7e)

![Screenshot 2024-03-25 184003](https://github.com/quarto-dev/quarto-cli/assets/50002526/741f679f-ee0a-4275-a09e-7144dea23b65)

I test this behavior using the standard template used in RStudio:

````qmd
---
title: "Untitled"
format: 
  html:
    embed-resources: true
editor: visual
---

## Quarto

Quarto enables you to weave together content and executable code into a finished document. To learn more about Quarto see <https://quarto.org>.

## Running Code

When you click the **Render** button a document will be generated that includes both content and the output of embedded code. You can embed code like this:

```{r}
1 + 1
```

You can add options to executable code like this

```{r}
#| echo: false
2 * 2
```

The `echo: false` option disables the printing of code (only output is displayed).
````

I recently updated Quarto and RStudio IDE. Here are my current settings:

- `quarto 1.5.25`
- `RStudio IDE "Ocean Storm" Release (4da58325, 2024-01-28) for windows`
- `Windows 11 Pro 23H2`

I'm trying to figure out if this issue is widespread or specific to my setup. Has anyone else encountered this?

**Comment by cscheid** [comment, marked as answer]
The .quarto folder stores project-specific files that need to persist from one render to another. In 1.5.25 that folder is being accidentally created for single-file renders, where it's not necessary. If you upgrade to 1.5.27 you will not see that behavior (I can't reproduce it on `main`).

**Reply by luifrancgom** [reply]
Thank you Carlos Scheidegger for your help 

As always the quarto team is fantastic where it is always helping the community

Thank you again and again and ... for your help

**Comment by jonathanto99** [comment]
As of Quarto 1.9.36, I am seeing the bug reappear for single file renders in Positron.
